tests/handlers/test_e2e_room_keys.py

```diff
@@ -78,7 +82,7 @@ def test_create_version(self) -> None:
                 },
             )
         )
-        self.assertEqual(res, "1")
+        self.assertEqual(version, "1")
```

Comment:
Slightly scary that what looks like a version number is a string!

Looks like this is deliberate though (:cry:): https://github.com/matrix-org/synapse/blob/d8cc86eff484b6f570f55a5badb337080c6e4dcd/synapse/storage/databases/main/e2e_room_keys.py#L451-L464

Reply:
Yeah, I don't know why they're strings, but 🤷